### DISCUSSION TITLE
Local development order

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#62] [Planning consideration]
# Local development order

Local development orders (LDOs) provide permitted development rights for specified types of development in defined locations.

### What we're looking for

We want to know more about:

* more about how local development orders are made, who makes them, what is the process, etc
* how do you use local development orders?
* what decisions are made using local development orders
* how would you like to use local development order data
* what data do you think should be included in any local development order data model
* examples of local development orders (see below)
* anything else we should know

**Comment by colmjude** [comment]
## Example local development orders

* [Leeds District Heating Network](https://www.leeds.gov.uk/planning/planning-policy/supplementary-planning-documents-and-guidance/local-development-order)

**Comment by psd** [comment]
The [Brownfield Passport: Making the Most of Urban Land](https://www.gov.uk/government/publications/planning-reform-working-paper-brownfield-passport/brownfield-passport-making-the-most-of-urban-land) policy paper, published 22 September 2024 contains proposals which:

> relate to the principle, the scale, and the form of development, and to the potential wider use of Local Development Orders to grant area-wide permissions – all in a way that retains appropriate local oversight.
